libraries/common/src/main/java/androidx/media3/common/VideoGraph.java

```diff
@@ -75,14 +75,13 @@ interface Listener {
    *
    * <p>If the method throws, the caller must call {@link #release}.
    *
-   * @return The id of the registered input, which can be used to get the underlying {@link
-   *     VideoFrameProcessor} via {@link #getProcessor(int)}.
+   * @param sequenceIndex The sequence index of the input which can aid ordering of the inputs.
```

Comment:
It seems it's easier if we require sequence indices start from 0, so please add something like "The index must start from zero"